content/en/events/2023/kcseu/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Contributor Summit Europe 2023"
+type: docs
+aliases: [ "/kcseu2023" , "/summit" ]
+weight: 1
+description: >
+  Kubernetes Contributor Summit Europe, hosted in Amsterdam Netherlands.
+
+---
+
+
+### About
+
+The Europe Summit will be a lightweight event taking place on May 16th
+in Amsterdam, The Netherlands alongside 
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" rel="noopener noreferrer" target="_blank">KubeCon</a>
+at the
+<a href="https://www.rai.nl/en" rel="noopener noreferrer" target="_blank">RAI Amsterdam</a>.
+
+The Summit will be a loosely organized event with a [relaxed schedule](/events/2023/kcseu/schedule),
+ including a Steering AMA, three hours of Unconference Sessions, 
+multiple SIG and WG meetings, and a Documentation Sprint.  This will be followed
```

Comment:
```suggestion
 including a Steering AMA, several hours of Unconference Sessions, 
potential SIG and WG meetings, and a Documentation Sprint.  This will be followed
```